src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -453,7 +460,8 @@ protected function renderCrumbs()
 				$href = "/{$href}";
 			}
 
-			$output .= $this->renderCrumb($crumb['name'], $href, $isLast);
+			$output .= $this->renderCrumb($crumb['name'], $href, $position, $isLast);
+			$position ++;
```

Comment:
I think whitespace before unary incrementation looks kinda weird - I've checked PSR and it doesn't seem to mention this, but could you do `$position++` anyway?